### DISCUSSION TITLE
[Help] change split border color

### DISCUSSION THREAD

**Discussion by edshamis** [catppuccin/nvim#443] [Q&A] [answered]
How to change color for horizontal and vertical splits? Especially for lazyvim setup, but any help is appreciated. Thanks 🙏

**Comment by rewhile** [comment, marked as answer]
See `:h hl-WinSeparator`

```lua
return {
  "catppuccin/nvim",
  name = "catppuccin",
  opts = {
    custom_highlights = function(C)
      return {
        WinSeparator = { fg = C.pink },
      }
    end,
  },
}
```

![image](https://user-images.githubusercontent.com/56817415/235294829-c444c2af-204a-45e7-8d2f-75a74c88c39b.png)

**Reply by Iamvismorf** [reply]
By any chance, what plugin do you use that shows the name of opened file in split buffers?
![image](https://github.com/user-attachments/assets/0ddb11a4-2868-4de2-977d-476056379bc7)

**Reply by vollowx** [reply]
that could be [dropbar.nvim](https://github.com/Bekaboo/dropbar.nvim)

**Reply by Iamvismorf** [reply]
Exactly what I am looking for. Thanks!
